dz17/package.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "dz17",
+  "private": "true",
+  "devDependencies": {
+    "@shcherbin/eslint-config": "^0.6.0",
+    "eslint": "^9.13.0"
+  }
+}
```

```suggestion
  },
  "type": "module"
}
```

нужно указать тип `module`